src/OrchardCore.Modules/OrchardCore.Settings/Controllers/AdminController.cs

```diff
@@ -40,6 +41,7 @@ public class AdminController : Controller
             H = h;
         }
 
+        [Admin("Settings/{groupId}")]
```

Comment:
This route is redirected to a alot, its worth giving it the old `AdminSettings` name.